datafusion/functions-aggregate/src/string_agg.rs

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-//! [`StringAgg`] and [`StringAggAccumulator`] accumulator for the `string_agg` function
+//! [`StringAgg`]  accumulator for the `string_agg` function
```

Comment:
```suggestion
//! [`StringAgg`] accumulator for the `string_agg` function
```
nit: remove an extra space